frontend/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
```

Comment:
remove ^, always lock to specific version type to avoid unexpected version changes :)